raiden-ts/src/config.ts

```diff
@@ -76,8 +71,6 @@ export const RaidenConfig = t.readonly(
       discoveryRoom: t.union([t.string, t.null]),
       additionalServices: t.readonlyArray(t.union([Address, t.string])),
       pfsMode: PfsModeC,
-      pfsRoom: t.union([t.string, t.null]),
```

Comment:
Do we need to do something for compatibility with old DBs?

Reply:
No, it's safe to have extra properties on the DB. The problem would be to change the format/type of something, or to add a required property which would not be there on already initialized DBs, only then we need a migration.